.github/workflows/check-devcontainer.yml

```diff
@@ -0,0 +1,42 @@
+# This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Devcontainer check
+
+on:
+  workflow_dispatch:
+  push:
+    # Run only on branches/commits and not tags
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "1 1 * * 1"
+
+
+jobs:
+  automated-tests:
+    runs-on: ubuntu-22.04
```

Comment:
We should perform test on our image, shouldn't we?

Reply:
We perform it on our image, this image is for the job itself, as host image for the runner, but tests are executed inside built devcontainer with our base image.